R/learn.R

```diff
@@ -52,6 +52,8 @@ dm_learn_from_db <- function(dest, ...) {
     return()
   }
 
+  dbname <- list(...)$dbname
```

Comment:
We could pull out the `dbname` argument explicitly, need to pass it with the ellipse in `db_learn_query()` .

Reply:
Sorry for the late reply.
I don't really understand what you mean by "pulling out".

There are currently 3 different arguments that can be given in the ellipsis: `dbname`, `schema` and `table_type`.

How (mabye `list(...)$dbname` isn't so nice?) and when should I access them?

Reply:
`dm_learn_from_db <- function(dest, dbname, ...)` and pass it explicitly to other functions.